boards/sparky_v2.hpp

```diff
@@ -272,7 +268,6 @@ static inline sparky_v2_conf_t sparky_v2_default_config()
     conf.pwm_config[2].timer                = TIM9;
     conf.pwm_config[2].rcc_timer            = RCC_TIM9;
     conf.pwm_config[2].channel              = Pwm_stm32::PWM_STM32_CHANNEL_2;
-    // max timer clock freq of TIM9 is not 84 but 168 MHz
```

Comment:
Don't you want to keep explanation of why the prescaler is not the same among different timer ?
To remind people that this is not an mistake